src/GenreBar.js

```diff
@@ -18,10 +18,23 @@ export default class GenreBar extends Component {
         })
         this.props.updateState({genresToFilter: checked})
     }
+    toggleGenres = () => {
+        if (this.state.showGenres) {
```

Comment:
Good implementation of conditional rendering, I think we could refactor this into a ternary and save a few lines.